ui/src/templates/containers/CommunityTemplatesIndex.tsx

```diff
@@ -34,14 +38,28 @@ interface StateProps {
   org: Organization
 }
 
-type Props = WithRouterProps & StateProps
+interface OwnProps extends WithRouterProps {
+  params: {templateName: string}
+}
+
+type Props = OwnProps & StateProps
 
 @ErrorHandling
-class CTI extends Component<Props> {
+class UnconnectedTemplatesIndex extends Component<Props> {
```

Comment:
what does `Unconnected` signal?

Reply:
https://react-redux.js.org/api/connect

we return `CommunityTemplatesIndex` as part of the public API of this. `UnconnectedTemplatesIndex` is just the plain React component that hasn't had redux injected into it.